src/main/scala/no/ndla/auth/service/TokenService.scala

```diff
@@ -32,7 +32,7 @@ trait TokenService {
         ("iat" -> now) ~
         ("exp" -> expires))
 
-      TokenResponse(JsonWebToken(jwtHeader, jwtClaims, clientSecret), "bearer", expires)
+      TokenResponse(JsonWebToken(jwtHeader, jwtClaims, clientSecret), "bearer", AuthProperties.TokenValidityInSeconds)
```

Comment:
Jeg får ikke kommentert på selve `TokenResponse` case classen, men beskrivelsen av `expires_in` er kanskje litt misvisende: "The number of seconds until this token expires". Får litt intrykk av at dette feltet blir automatisk oppdatert på ett eller annet sted, som vel ikke stemmer helt?
Kanskje i stedet noe som: "The lifetime in seconds since token creation"?
https://tools.ietf.org/html/rfc6749#section-4.2.2